common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -529,18 +529,18 @@ def ignore_conditions_table
 
         # Filter out the conditions where from_config_file is false and dependency is in @dependencies
         valid_ignore_conditions = @ignore_conditions.select do |ic|
-          !ic[:from_config_file] && dependencies.any? { |dep| dep.name == ic[:dependency_name] }
+          ic["source"] == "@dependabot ignore command" && dependencies.any? { |dep| dep.name == ic["dependency-name"] }
```

Comment:
❓ Would it make sense to make this 🤏 more broad? It seems unlikely to be falsely triggered, and using a Regex could make this a little more resilient to change. Probably YAGNI, though.

```ruby
ic["source"] =~ /@dependabot ignore/
```

Reply:
Yeah I suppose we could do that